spec/requests/settings/main_spec.rb

```diff
@@ -49,7 +48,7 @@
 
       within_section "User details", section_element: :section do
         validation_message = find_field("Email").native.attribute("validationMessage")
-        expect(validation_message).to eq "Please fill out this field."
+        expect(validation_message).to eq "Please fill in this field."
```

Comment:
Why was this change needed?

Reply:
when i tested locally the text was coming as `fill in` - changing it back now could be a local setup issue